### PR TITLE
Created proposed  LICENSE

### PR DESCRIPTION
**Pull request by frederik256**
Your project did not come with a license, so I added three clause license for your consideration (with copyright in your name). 
The lack of a license creates a problem for adoption in contexts where open source usage policies exist.

### CODE REVIEW

**Comment by frederik256** [comment]
Your project did not come with a license, so I added three clause license for your consideration (with copyright in your name).
The lack of a license creates a problem for adoption in contexts where open source usage policies exist.

**Comment by jack482653** [comment]
@frederik256 Thanks for your pull request :).

Cloud you also change the license section in the package.json? Currently it is `ISC` .

**Comment by frederik256** [comment]
done.

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "test": "mocha"
   },
   "author": "Kevin Zhuang",
-  "license": "ISC",
+  "license": "BSD",
```
Comment (jack482653) [inline package.json]: According to[ package.json document](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#license) and [the full list of SPDX license IDs](https://spdx.org/licenses/), it should be `BSD-3-Clause`

Comment (frederik256) [reply package.json]: updated accordingly.

**Comment by jack482653** [comment]
LGTM

**Pull request merged**